### PR TITLE
ROX-27138: add GitHub container registry

### PR DESCRIPTION
**Pull request by stehessel**
Version(s): 4.7

Issue: [ROX-27138](https://issues.redhat.com/browse/ROX-27138)

[Link to docs preview
](https://86008--ocpdocs-pr.netlify.app/openshift-acs/latest/integration/integrate-with-image-registries.html#manual-configuration-image-registry-ghcr_integrate-with-image-registries)

QE review: **ACS has no QE, approved by SME**
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by kcarmichael08 — COMMENTED** [review]
Couple of small issues - otherwise, lgtm!

File: modules/manual-configuration-image-registry-ghcr.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+//
+// * integration/integrate-with-image-registries.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="manual-configuration-image-registry-ghcr_{context}"]
+= Manually configuring GitHub Container Registry
+
+You can integrate {product-title} with GitHub Container Registry (GHCR).
+
+.Prerequisites
+* You need a GitHub account and personal access token with at least `packages:read` permissions. See [Working with the Container registry
```
[the remark this answers is not in the document]
Comment (stehessel) [reply modules/manual-configuration-image-registry-ghcr.adoc:11]: should be fixed

**Review by kcarmichael08 — COMMENTED** [review]
Hi, just need to fix the link - if you look at the preview it's still not quite right.

**Review by kcarmichael08 — COMMENTED** [review]
LGTM. I see that the module has been inserted between the Quay content and the Quay additional resource section, but I will fix that in a separate PR.

**Pull request merged**